samples/ClientSample/RawSample.cs

```diff
@@ -26,7 +26,7 @@ internal class RawSample
                 args = args.Except(new[] { "--debug" }).ToArray();
             }
 
-            var baseUrl = "http://localhost:5000/chat";
```

Comment:
Hmm, why did this change? It should be the default port when starting the SocketsSample up in Kestrel. IIS Express will use a random port, but using Kestrel should have a consistent port number.

Alternatively, we could just drop the default and require a URL on the command line :)